spec/unit/actions/shared_examples_for_user_actions.rb

```diff
@@ -24,3 +24,20 @@
     subject.class.call(params)
   end
 end
+
+shared_examples 'when using unsuported sync_mode' do
+  before do
```

Comment:
Identical code found in shared_examples_for_user_actions.rb:29, synchronize_users_spec.rb:165 (mass = 180)